HOLD FOR RELEASE: Adds adoption rate docs (#1052)
replicatedhq/replicated-docs: docs/vendor/customer-adoption.md

![Adoption rate section on Customers page ](/images/customer_adoption_rates.png)

[View a larger version of this image](/images/customer_adoption_rates.png)

As shown in the screenshot above, the **Adoption rate** section includes a graph and key adoption rate metrics. For more information about how to interpret this data, see [Adoption Rate Graph](#graph) and [Adoption Rate Metrics](#metrics) below.

The **Adoption rate** section also displays the number of customers assigned to the selected channel and a link to the report that you can share with other members of your team.

You can filter the graph and metrics in the **Adoption rate** section by:
* License type (Paid, Trial, Dev, or Community)
* Time period (the previous month, three months, six months, or twelve months).
* Release channel where the instance license is assigned, such as Stable or Beta.

## Adoption Rate Graph {#graph}

The **Adoption rate** section includes a graph that shows the percent of active instances that are running different versions of your application on each day within the selected time period.

The following shows an example of an adoption rate graph with three months of data:

> ![Adoption rate graph showing three months of data](/images/adoption_rate_graph.png)

Author: @dexhorthy I was thinking it could be useful to show the popup that appears when you hover over the graph as part of this screenshot, but for whatever reason I wasn't able to access the wordpress app that you shared with me. (I don't think this is a blocker for release or anything)

Reviewer: I just reinvited `paige+wpe@replicated.com` to a demo account

If there are no active instances of your application, then the adoption rate graph displays a "No Instances" message.

## Adoption Rate Metrics {#metrics}

The **Adoption rate** section also includes metrics that are key indicators of how frequently your customers discover and complete upgrades to new versions of your application.

The following shows an example of the metrics in the **Adoption rate** section:

![Adoption rate metrics showing](/images/adoption_rate_metrics.png)

[View a larger version of this image](/images/adoption_rate_metrics.png)

As shown in the image above, the **Adoption rate** section displays the following metrics:
* Instances on last three versions
* Unique versions
* Median relative age
* Upgrades completed

Based on the time period selected, each metric includes an arrow that shows the change in value compared to the previous period. For example, if the median relative age today is 68 days, the selected time period is three months, and three months ago the median relative age was 55 days, then the metric would show an upward-facing arrow with an increase of 13 days. 

> The following table describes each metric in the **Adoption rate** section, including the formula used to calculate its value and the recommended trend for the metric over time: 

Author: I added a column in this table for "Target Trend" to explain the target change in time for each metric.

> # Adoption Rate (Beta)

Reviewer: ```suggestion
# Adoption Report (Beta)
```

> This topic describes the insights in the **Adoption rate** section on the Replicated vendor portal **Customers** page.

Reviewer: ```suggestion
This topic describes the insights in the **Adoption** section on the Replicated vendor portal **Customers** page.
```

:::note

> The **Adoption Report** section is currently in Beta -- you will see it labeled as such in the Vendor Portal. the character of this data and how it is computed and displayed is subject to change.

Reviewer: ```suggestion
The **Adoption Report** section is currently in Beta -- you will see it labeled as such in the Vendor Portal. The character of this data, how it is computed, and how it is displayed is subject to change.
```

> :::

Author: @dexhorthy to correct this notice with the actual name of the feature flag that they should request

## About Adoption Rate

> The **Adoption rate** section on the **Customers** page provides insights about the rate at which your customers upgrade their instances and adopt the latest versions of your application. As an application vendor, you can use these adoption rate metrics to learn if your customers are completing upgrades regularly, which is a key indicator of the discoverability and ease of application upgrades.

Reviewer: ```suggestion
The **Adoption** section on the **Customers** page provides insights about the rate at which your customers upgrade their instances and adopt the latest versions of your application. As an application vendor, you can use these adoption rate metrics to learn if your customers are completing upgrades regularly, which is a key indicator of the discoverability and ease of application upgrades.
```

> The vendor portal generates adoption rate data from all of your customer's active application instances. An instance is considered _active_ if no more than two weeks have passed since its last check-in event. For more information about instance check-ins, see [How the Vendor Portal Collects Instance Data](instance-insights-event-data#about-reporting) in _About Instance and Event Data_.

Reviewer: @divolgin is this note about "active" instances true? Or do we only collect data from instances that checked in within the period? My guess is that the language here can be simplified a bit.

Reviewer 2: We show data for instances that checked in the given time period.

The adoption rates on the channel cards themselves only show instances active within the last 24 hours.

Author: Gotcha, so as long as they had a check-in one time in the given time period. Agreed Dex I can simplify this (not worth defining active here).

> The following screenshot shows an example of the **Adoption rate** section:

Reviewer: ```suggestion
The following screenshot shows an example of the **Adoption** report:
```

![Adoption rate section on Customers page ](/images/customer_adoption_rates.png)

[View a larger version of this image](/images/customer_adoption_rates.png)

> As shown in the screenshot above, the **Adoption rate** section includes a graph and key adoption rate metrics. For more information about how to interpret this data, see [Adoption Rate Graph](#graph) and [Adoption Rate Metrics](#metrics) below.

Reviewer: ```suggestion
As shown in the screenshot above, the **Adoption** report includes a graph and key adoption rate metrics. For more information about how to interpret this data, see [Adoption Rate Graph](#graph) and [Adoption Rate Metrics](#metrics) below.
```

> The **Adoption rate** section also displays the number of customers assigned to the selected channel and a link to the report that you can share with other members of your team.

Reviewer: ```suggestion
The **Adoption** report also displays the number of customers assigned to the selected channel and a link to the report that you can share with other members of your team.
```

> You can filter the graph and metrics in the **Adoption rate** section by:

Reviewer: ```suggestion
You can filter the graph and metrics in the **Adoption** report by:
```

* License type (Paid, Trial, Dev, or Community)
* Time period (the previous month, three months, six months, or twelve months).
* Release channel where the instance license is assigned, such as Stable or Beta.

## Adoption Rate Graph {#graph}

> The **Adoption rate** section includes a graph that shows the percent of active instances that are running different versions of your application on each day within the selected time period.

Reviewer: ```suggestion
The **Adoption** report includes a graph that shows the percent of active instances that are running different versions of your application on each day within the selected time period.
```

The following shows an example of an adoption rate graph with three months of data:

![Adoption rate graph showing three months of data](/images/adoption_rate_graph.png)

[View a larger version of this image](/images/adoption_rate_graph.png)

> As shown in the image above, the graph plots the number of active instances on each day in the selected time period. Additionally, that graph uses colors to represent the application versions that were active on each day. The key to the left of the graph shows the unique color that is assigned to each application version. You can use this color-coding to see at a glance the percentage of active instances that were running different versions of your application across the selected time period.

Reviewer: ```suggestion
As shown in the image above, the graph plots the number of active instances in each week in the selected time period. Additionally, the graph uses colors to represent the application versions that were active in each week. The key to the left of the graph shows the unique color that is assigned to each application version. You can use this color-coding to see at a glance the percentage of active instances that were running different versions of your application across the selected time period.
```

> You can also hover over a color-coded section in the graph to view the number and percentage of active instances that were running the version on the given day.

Reviewer: ```suggestion
You can also hover over a color-coded section in the graph to view the number and percentage of active instances that were running the version in a given period.
```

> # Adoption Rate

Author: All new content to describe adoption rate data in its own topic.

If there are no active instances of your application, then the **Adoption rate** section displays a "No Instances" message.

## Adoption Rate Metrics

The **Adoption rate** section also includes metrics that are key indicators of how frequently your customers discover and complete upgrades to new versions of your application.

**SCREENSHOT OF METRICS SECTION**

As shown in the screenshot above, the **Adoption rate** section displays the following metrics:
* Instances on last three versions
* Unique versions
* Median relative age
* Upgrades completed

Based on the time period selected, the **Adoption rate** section also displays arrows with the change in the value of each metric compared to the previous period. For example, if the median relative age today is 68 days, the selected time period is three months, and three months ago the median relative age was 55 days, then the metric would show an upward-facing arrow with an increase of 13 days. 

The following table describes each metric in the **Adoption rate** section, including the formula used to calculate its value and the recommended target value: 

<table>
  <tbody>
    <tr>
      <th>Metric</th>
      <th>Description</th>

>       <th>Recommended Target Value</th>

Author: Testing out a column here where we describe the recommended target value that they should be shooting for. I thought this might be useful info, and it would also then imply which arrow direction is "good/bad".

Just wasn't sure if "target value" is something we're prepared to offer

Author: Perhaps we could also add more general suggestions, like the lower/higher the better
